### ISSUE TITLE
Training details for Finetuning

### ISSUE THREAD

**Issue by Scipio1996** [RL4M/PCRLv2#2]
Dear authors:

This is great work, which makes me learn much.
However, I find some differences between the paper and the source code.
Specifically, in your paper, you said the initial rate is 1e-2, while in the code, the learning rate is 1e-3.
Which is right.

Thank you in advance

**Comment by funnyzhou** [comment]
Hi,

This might be a typo in the manuscript. You should use 1e-3 as suggested by the codebase.

**Comment by liaochuanlin** [comment]
@Scipio1996 Excuse me, have you reproduced the data set of the brain?
